src/Teams/Data/Repositories/IProgramCodeQuestionRepository.cs

```diff
@@ -9,5 +9,7 @@ namespace Teams.Data.Repositories
     public interface IProgramCodeQuestionRepository
     {
         ProgramCodeQuestion PickById(Guid id);
+        void SaveProgramText(ProgramCodeQuestion question, string text);
+        void AddToDb(ProgramCodeQuestion question);
```

Comment:
Is this method used anywhere? Do you have additional files that were not included in the pull request? "ToDb" looks redundant. It's better to use just "Add".

Reply:
I used it to create a new db by adding some test entities. We could leave this method to create new program code questions in future. But if you think we won't need it more, I'll remove it with the next commit.

Reply:
Okay, let's keep it

Reply:
Ok, I've updated it